src/main/flight/feedforward.c

```diff
@@ -70,88 +67,96 @@ FAST_CODE_NOINLINE float feedforwardApply(int axis, bool newRcFrame, feedforward
         float rcCommandDelta = getRcCommandDelta(axis);
         float setpoint = getRawSetpoint(axis);
         const float rxInterval = getCurrentRxRefreshRate() * 1e-6f;
-        const float rxRate = 1.0f / rxInterval;
+        const float rxRate = 1.0f / rxInterval; // eg 150 for a 150 RC link
```

Comment:
Does the '150 RC link' stand for '150 Hz RC link'? Could probably be clarified.

Reply:
Yes, sorry.  I've clarified as suggested, and squashed.